MA-Bylaws.md

```diff
@@ -435,27 +421,19 @@ The proposal of a project for registration as MAP must contain
   - Initial members of the project (individual and organizational). A
     MAP must have a reasonable number of members.
 
-  - Name, organization and position of the project leader.
-
-The project leader must be a member of the MA before the voting about
-the project's registration. If this is not already the case, the
-procedure of § 3 can be applied. If the project leader is no longer a
-member of MA, another MA member has to be elected as project leader
-according to the project rules. Until a new project leader has been
-elected, the MA Chairperson is the leader of such a project. After the
-MAP registration, the project leader can apply for approval as MA Board
-member by the MA.
-
-For clarification: other projects members need not be MA members.
+  - Name, organization and position of the project leader (and deputy) fulfilling 
+the requirements of §23.
 
 ## § 23 Terms and conditions for Modelica Association Projects ##
 
+An MA project leader and the deputy needs to fulfil the requirements for 
```

Comment:
It is too late to change the wording, but this should ready "An MA project leader and the deputy need to fulfil" since, joined by a conjunction here, we are talking about more than one person (_e.g.,_ consider the same phrase using a pronoun: "they need to fulfil").  Alternatively, one could have said "Every MA Project Leader and Deputy Project Leader needs to fulfil".  It is also a bit odd to use "an" at the start and "the" later.  Presumably this was meant to convey "An MA project leader and their deputy" (_i.e.,_ the two are associated in some way) but that actually doesn't matter in this context.  Whether they are associated does not change any of the requirements.

Reply:
As we do not change the semantics but only compliance with English grammar, for me a last minute change would be possible before the voting. @MartinOtter : what do you think?
(in the future it would be good to get such feedback earlier)